_events/sr2017/southampton-tech-day-december.md

```diff
@@ -12,7 +12,14 @@ The Southampton tech days take advantage of the facilities of the University of
 
 ## Directions
 
-This years tech day will be at the University of Southampton, the exact location will be announced soon!
+The location of our tech day is on a different campus to that of Kickstart, please check our map!
```

Comment:
Why is this comment present only on this event? It appears that it would apply to the other ones as well.

Minor: perhaps "The location of this tech day" (swaps "our" for "this") would be clearer here?
Alternatively, is this meant to be "our tech days" (plural)? If so it makes the assumption that people coming to later tech days will have been to earlier ones, which is a poor assumption IMO. In the interests of clarity it should be on the other event pages as well, even if phrased as a general comment ("our tech days").